CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+<!-- omit in toc -->
+# Contributing to Fluxnet Open Source Code Committee Resource Guide
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
+
+> And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
+> - Star the project
+> - Tweet about it
+> - Refer this project in your project's readme
+> - Mention the project at local meetups and tell your friends/colleagues
+
+<!-- omit in toc -->
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [I Have a Question](#i-have-a-question)
+- [I Want To Contribute](#i-want-to-contribute)
+- [Suggesting Enhancements](#suggesting-enhancements)
+- [Reporting Bugs](#reporting-bugs)
+- [Your First Code Contribution](#your-first-code-contribution)
```

Comment:
These sections aren't in the guide yet (but I also don't think we need all of them now)?

Probably we should have an "improving the documentation" section (but maybe call it "improving the website"?) with instructions about how to put a PR for edits to the website, for minor edits (i.e., typos, broken links, etc) and major edits (adding new content). 

In the website I activated the links in the top menu to edit pages (so hopefully it will be easy for people to make suggested edits if they see a problem). Then we could make sure this contribution guide shows up on the website also?

But we could do these changes in a separate PR if you prefer.